alephium/contracts/tests/get_token.ral

```diff
@@ -1,3 +1,3 @@
-TxScript GetToken(sender: Address, token: TestToken, amount: U256) {
-    token.buy{sender -> ALPH: amount}(sender, amount)
+TxScript GetToken(sender: Address, token: TestToken, amount: U256, factor: U256) {
+    token.buy{sender -> ALPH: amount}(sender, amount, factor)
```

Comment:
Because this is just for testing, the purpose of adding the factor parameter at the beginning is not to hard-code the conversion ratio of ALPH and the token in the contract.